src/app/features/management/group-management-view/edit-group-view/edit-group-view.component.ts

```diff
@@ -0,0 +1,230 @@
+      },
+    });
+  }
+
+  getUserById(id: string): Observable<User> {
+    return this.userService.getUserById(id);
+  }
+
+  getOperationById(id: string): Observable<Operation> {
+    return this.operationService.getOperationById(id);
+  }
+
+  searchOperation(query: string): Observable<Operation[]> {
+    if (query === '') {
+      return of([]);
+    }
+    return this.operationService.searchOperations({
+        query: query,
+        limit: 5,
+        offset: 0,
+      }, {},
+    ).pipe(
+      map((res: SearchResult<Operation>): Operation[] => {
+        return res.hits;
+      }));
+  }
+
+  searchUser(query: string): Observable<User[]> {
+    if (query === '') {
+      return of([]);
+    }
+    return this.userService.searchUsers({
+      query: query,
+      limit: 5,
+      offset: 0,
+    }, true).pipe(
+      map((res: SearchResult<User>): User[] => {
+        return res.hits;
+      }));
+  }
+
+  asUser(entity: User): User {
+    return entity as User;
+  }
+
+  asOperation(entity: Operation): Operation {
+    return entity as Operation;
+  }
+
+  /**
+   * Goes back to 'groups/'.
+   */
+  close() {
+    this.router.navigate(['..'], { relativeTo: this.route }).then();
+  }
+
+  /**
+   * Removes a member.
+   * @param memberId
+   */
+  removeMember(memberId: string): void {
+    this.form.controls.members.patchValue(this.form.controls.members.value.filter(item => item != memberId));
+  }
+
+  private static mapOrderBy(s: string): UserSort | undefined {
+    switch (s) {
+      case 'lastName':
+        return UserSort.ByLastName;
+      case 'firstName':
+        return UserSort.ByFirstName;
+      case 'username':
+        return UserSort.ByUsername;
+      case '':
+        return undefined;
+      default:
+        throw new MDSError(MDSErrorCode.AppError, `unsupported order-by: ${ s }`);
+    }
+  }
+
+  /**
+   * Called when sort changes.
+   * @param sort New sort.
+   */
+  sortChange(sort: Sort): void {
+    let sortedMembers = this.members.map(x => x);
+    switch (EditGroupViewComponent.mapOrderBy(sort.active)) {
+      case UserSort.ByLastName:
+        if (orderDirFromSort(sort) === OrderDir.Asc) {
+          sortedMembers.sort((a, b) => a.lastName.localeCompare(b.lastName));
+        } else {
+          sortedMembers.sort((a, b) => b.lastName.localeCompare(a.lastName));
+        }
+        break;
+      case UserSort.ByFirstName:
+        if (orderDirFromSort(sort) === OrderDir.Asc) {
+          sortedMembers.sort((a, b) => a.firstName.localeCompare(b.firstName));
+        } else {
+          sortedMembers.sort((a, b) => b.firstName.localeCompare(a.firstName));
+        }
+        break;
+      case UserSort.ByUsername:
+        if (orderDirFromSort(sort) === OrderDir.Asc) {
+          sortedMembers.sort((a, b) => a.username.localeCompare(b.username));
+        } else {
+          sortedMembers.sort((a, b) => b.username.localeCompare(a.username));
+        }
```

Comment:
lol wenn du sowas schon 3 mal copy pastest und das nicht, weil es was unabhängiges tut, dann erstell dir doch in `util.ts` oder so eine helper function dafür. ist vermutlich auch nicht das letzte mal, dass wir sowas brauchen.